### PR TITLE
Add STM32F1xx support for TFTGLCD

### PR DESCRIPTION
**Pull request by Serhiy-K**
- Add STM32F1xx support for TFTGLCD
- Add STM32F1xx support for STM32F103C Bluepill dev board
- Small fix for edit first selected point in UBL for TFTGLCD

### Benefits

SD support can be added to STM32F103CB Bluepill dev board with STM32 Base Environment

### CODE REVIEW

File: Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp [hunk 1]
```diff
@@ -972,7 +972,14 @@
 
       MeshFlags done_flags{0};
       const xy_int8_t &lpos = location.pos;
-      do {
+
+      #if IS_TFTGLCD_PANEL
+        lcd_mesh_edit_setup(0);                             // Change current screen before calling ui.ubl_plot
```
Comment (rhapsodyv) [inline Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp:977]: Is this really needed? Just to avoid adding specific UI code in general code.

**Comment by Serhiy-K** [comment]
This line is needed because **lcd_mesh_edit_setup** changes the current screen to **_lcd_mesh_fine_tune** on the first call, but **ui.ubl_plot** does not. Without this line, editing the first selected point will look like this
Marlin-Edit_param-V3
that is, one line on a black background and for other points like this
Marlin-UBL_Mesh_Editor
With this line, editing all points looks like in the second picture.

**Comment by rhapsodyv** [comment]
Why this problem don't happen on the another UIs? 🤔

**Comment by Serhiy-K** [comment]
Because other UIs edit all points as a simple edit line as in the first picture, not in the ubl_plot screen. For text screens like 20x4 this looks normal, but on graphic screens it is looks not very nice and not informative.

**Pull request merged**